bigbluebutton-html5/package.json

```diff
@@ -2,16 +2,20 @@
   "name": "bbb-html5-client",
   "description": "BigBlueButton HTML5 Client",
   "scripts": {
-    "start": "cd app;./start.sh",
-    "preinstall": "npm install grunt-cli",
+    "start": "if test \"$NODE_ENV\" = \"production\" ; then npm run start:prod; else npm run start:dev; fi",
+    "start:prod": "meteor reset && if test $npm_config_rootURL; then npm run start:prod:customRootURL; else npm run start:prod:defaultRootURL; fi",
+    "start:prod:customRootURL": "if test $npm_config_prodHome; then HOME=$npm_config_prodHome ROOT_URL=$npm_config_rootURL NODE_ENV=production meteor; else ROOT_URL=$npm_config_rootURL NODE_ENV=production meteor; fi",
+    "start:prod:defaultRootURL": "if test $npm_config_prodHome; then HOME=$npm_config_prodHome ROOT_URL=http://127.0.0.1/html5client NODE_ENV=production meteor; else ROOT_URL=http://127.0.0.1/html5client NODE_ENV=production meteor; fi",
```

Comment:
Could you please add `meteor --production` flag to both "start:prod" scripts above?
Minifying js and css will help simulate production mode locally.